js/stable.js

```diff
@@ -808,7 +809,7 @@ var theSort =
 		if(y==null) y = "";
 		var a = (x + "").toLowerCase();
 		var b = (y + "").toLowerCase();
-		return((a < b) ? -1 : (a > b) ? 1 : 0);
+		return((a.localeCompare(b) < b.localeCompare(a)) ? -1 : (a.localeCompare(b) > b.localeCompare(a)) ? 1 : 0);
```

Comment:
Why you need such strange constuction? Try to read about function localeCompare, please. About its return value.

Reply:
Oh yeah my bad... ;)